app/assets/stylesheets/reset.scss

```diff
@@ -123,6 +123,10 @@ input, textarea, button, select {
   vertical-align: baseline;
   font-family: inherit; }
 
+select {
```

Comment:
Is this necessary? Why is this added? How does this affect to all other `select` elements? Can we avoid adding this?